### PR TITLE
Correctly parse Course API response.

### PR DESCRIPTION
**Pull request by peter-fogg**
@clintonb @fredsmith

### CODE REVIEW

File: course_discovery/apps/courses/models.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def refresh_all_course_api_data(cls, access_token):
                 page = None
 
             for body in results:
-                Course(body['id']).update(body)
+                Course(body['course_id']).update(body)
```
Comment (clintonb) [inline course_discovery/apps/courses/models.py:200]: Are we looking at the same API? Should be https://courses.stage.edx.org/api/courses/v1/courses/?username=clintonb.

Comment (clintonb) [inline course_discovery/apps/courses/models.py:200]: Actually, both fields are correct.

Comment (peter-fogg) [inline course_discovery/apps/courses/models.py:200]: I think we're not looking at the same API... https://courses-loadtest-edx.edx.org/api/courses/v1/courses/?page=1&page_size=50&username=ecommerce_worker

Comment (peter-fogg) [inline course_discovery/apps/courses/models.py:200]: It looks like edx/edx-platform@a00c2a7c4786c879615b813b591f679169015bfd deprecated `course_id` in favor of `id`. I'm guessing that loadtest has an older version of platform.

Comment (clintonb) [inline course_discovery/apps/courses/models.py:200]: Wow! I wonder how many teams have run load tests against this older version of LMS. Would you build an AMI and ask @fredsmith to deploy, please?

**Comment by peter-fogg** [comment]
Closing this in favor of updating platform.

**Pull request closed without merging**